spec/features/resource_manager_manages_schedules_spec.rb

```diff
@@ -179,22 +170,22 @@ def and_the_guider_has_those_time_slots_available
   def and_they_delete_the_schedule
     @page = Pages::EditUser.new
     @page.load(id: @guider.id)
-    @page.delete.click
+    @page.schedules.first.delete.click
   end
 
   def and_the_schedule_is_deleted
     expect(@guider.reload.schedules).to be_empty
   end
 
-  def then_they_cant_delete_the_schedule
+  def then_they_cant_edit_the_schedule
     @page = Pages::EditUser.new
     @page.load(id: @guider.id)
-    expect(@page.delete).to be_disabled
+    expect(@page.schedules.first).to_not have_edit
```

Comment:
This is probably not the behaviour you want. I think you mean: `.to have_no_edit`. They are quite different semantically.